### PR TITLE
Bump to 3.06.1 and add unix

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge/nsis, this is ready.

**Comment by jakirkham** [comment]
Do you have a use case for this on UNIX? My understanding is this is primarily used to create installers on Windows.

**Comment by isuruf** [comment]
You can create installers for windows from unix.

**Comment by mbargull** [comment]
@isuruf, thanks for the update and Unix addition and also for finally modernizing this to the `conda-build`+`libarchive` era!

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+pushd plugins
+cp "UAC/U/UAC.dll" "$PREFIX_NSIS/Plugins/x86-unicode/"
+cp "untgz/Plugins/x86-unicode/untgz.dll" "$PREFIX_NSIS/Plugins/x86-unicode/"
+cp "UnicodePathTest/Plugin/UnicodePathTest.dll" "$PREFIX_NSIS/Plugins/x86-unicode/"
+popd
+
+cd src
+scons \
+  CC="${CC}" CXX="${CXX}" APPEND_CCFLAGS="${CXXFLAGS}" APPEND_LINKFLAGS="${LDFLAGS}" \
```
Comment (mbargull) [inline recipe/build.sh:17]: Re `APPEND_CCFLAGS="${CXXFLAGS}"`: yes, this means `CXXFLAGS` will also get used for the C compilations, so not perfect either. But I found it a bit clearer than adding `$BUILD_PREFIX/bin/gcc`.

**Comment by mbargull** [comment]
Linux build fails because it says the patch could not be applied.
Either the line endings are incorrectly checked out in the CI or it's maybe a bug in `conda-build>=3.20`.
Locally the build with `conda-build=3.19.2` works fine and the line endings in the repository https://raw.githubusercontent.com/conda-forge/nsis-feedstock/8958bc2b1ac42279d1950b2e47980c52020db260/recipe/0001-Always-use-relative-paths-for-CONF-and-DATA-directories.patch are also correct.

**Comment by mbargull** [comment]
>  or it's maybe a bug in `conda-build>=3.20`.

Yep, updating to `conda-build=3.20.2` breaks the build locally.

**Comment by mbargull** [comment]
> > or it's maybe a bug in `conda-build>=3.20`.
> 
> Yep, updating to `conda-build=3.20.2` breaks the build locally.

So, not a bug, actually. The line endings were correct, but I unintentionally replaced tabs with spaces in the patch :facepalm:.

**Comment by mbargull** [comment]
@isuruf, do the changes I added look okay to you?

**Review by mbargull — APPROVED** [review]

**Comment by isuruf** [comment]
Looks good to me. Thanks for the updates

**Comment by mbargull** [comment]
And thanks to you!

**Pull request merged**